meinberlin/apps/contrib/templatetags/contrib_tags.py

```diff
@@ -115,3 +118,24 @@ def get_proper_elided_page_range(p, number, on_each_side=1, on_ends=1):
     return paginator.get_elided_page_range(
         number=number, on_each_side=on_each_side, on_ends=on_ends
     )
+
+
+@register.simple_tag
+def get_external_footer():
+    # The BO footer needs to be included here as we can't include external
+    # html in the browser. Add a cron job to update the file periodically via
+    # the get_footer management command.
```

Comment:
We can have a scheduled celery task instead of a management command. the task can be called also inline directly as a simple function below in line 134.

Reply:
While I agree, this will need work on the admin site as we currently don't have celery beat running or configured, so maybe this is out of scope of thids PR? We already have everything to run a management command as a cron job in a single line. So I would suggest to have a separate PR which converts periodic tasks to celery beat? What do you think @m4ra @philli-m

Reply:
@goapunk @m4ra it's a bit out of my area so happy to go with whatever you guys think. This is separate to the path fix though right?
However we choose to do it, it should be replicated in GD as that's where I got this code from